I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 30, in script
    $playerBag = Inventory()
  File "game/script.rpy", line 30, in <module>
    $playerBag = Inventory()
NameError: name 'Inventory' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 30, in script
    $playerBag = Inventory()
  File "C:\Users\Shannon\Desktop\renpy-6.99.11-sdk\renpy\ast.py", line 805, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Shannon\Desktop\renpy-6.99.11-sdk\renpy\python.py", line 1641, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 30, in <module>
    $playerBag = Inventory()
NameError: name 'Inventory' is not defined

Windows-8-6.2.9200
Ren'Py 6.99.11.1749
Div III 1.0
